clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -165,7 +165,8 @@ public String toString() {
             return "(error=" + error + ", highWaterMark=" + highWatermark +
                     ", lastStableOffset = " + lastStableOffset +
                     ", logStartOffset = " + logStartOffset +
-                    ", abortedTransactions = " + abortedTransactions + ", records=" + records + ")";
+                    ", abortedTransactions = " + abortedTransactions +
+                    ", recordsSizeInBytes=" + records.sizeInBytes() + ")";
```

Comment:
Yeah, I meant `FetchResponse` indeed. One of those cases where I thought one thing and wrote another.